mysql-compatibility.md

```diff
@@ -93,171 +76,118 @@ mysql> select _tidb_rowid, id from t;
-
-{{< copyable "sql" >}}
-
-```sql
-CREATE TABLE t1 (a INT) ENGINE=MyISAM;
-```
-
-```
-Query OK, 0 rows affected (0.14 sec)
-```
-
-{{< copyable "sql" >}}
-
-```sql
-SHOW CREATE TABLE t1;
-```
+For compatibility reasons, TiDB supports the syntax to create tables with alternative storage engines. In actual implementation, TiDB describes the metadata as InnoDB storage engines.
 
-```
-*************************** 1. row ***************************
-       Table: t1
-Create Table: CREATE TABLE `t1` (
-  `a` int(11) DEFAULT NULL
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
-1 row in set (0.00 sec)
-```
-
-Architecturally, TiDB does support a similar storage engine abstraction to MySQL, and user tables are created in the engine specified by the [`--store`](/command-line-flags-for-tidb-configuration.md#--store) option used when you start tidb-server (typically `tikv`).
+Architecturally, TiDB supports a similar storage engine abstraction to MySQL, and user tables are created in the engine specified by the [`--store`](/command-line-flags-for-tidb-configuration.md#--store) option used when you start TiDB server.
 
 ### SQL modes
 
-TiDB supports **all of the SQL modes** from MySQL 5.7 with minor exceptions:
-
-- The compatibility modes deprecated in MySQL 5.7 and removed in MySQL 8.0 are not supported (such as `ORACLE`, `POSTGRESQL` etc).
-- The mode `ONLY_FULL_GROUP_BY` has minor [semantic differences](/functions-and-operators/aggregate-group-by-functions.md#differences-from-mysql) to MySQL 5.7, which we plan to address in the future.
-- The SQL modes `NO_DIR_IN_CREATE` and `NO_ENGINE_SUBSTITUTION` are supported for compatibility, but are not applicable to TiDB.
-
-### Version-specific comments
-
-TiDB executes all MySQL version-specific comments, regardless of the version they apply to. For example, the comment `/*!90000 */` would instruct a MySQL server less than 9.0 to not execute code. In TiDB this code will always be executed:
-
-```sql
-mysql 8.0.16> SELECT /*!90000 "I should not run", */ "I should run" FROM dual;
-+--------------+
-| I should run |
-+--------------+
-| I should run |
-+--------------+
-1 row in set (0.00 sec)
-
-tidb> SELECT /*!90000 "I should not run", */ "I should run" FROM dual;
-+------------------+--------------+
-| I should not run | I should run |
-+------------------+--------------+
-| I should not run | I should run |
-+------------------+--------------+
-1 row in set (0.00 sec)
-```
-
-### Lock tables
-
-Support for `LOCK TABLE` syntax is currently experimental, and must be explicitly enabled ([TiDB #10343](https://github.com/pingcap/tidb/pull/10343)).
-
-It is recommended to use the historical reads feature of `tidb_snapshot` to produce consistent reads, instead of `FLUSH TABLES WITH READ LOCK`. Support for `tidb_snapshot` is available in [Mydumper](/mydumper-overview.md)).
+- The compatibility modes deprecated in MySQL 5.7 and removed in MySQL 8.0 are not supported (such as `ORACLE` and `POSTGRESQL`).
```

Comment:
- Does not support the compatibility mode, such as `ORACLE` and `POSTGRESQL`. The compatibility mode is deprecated in MySQL 5.7 and removed in MySQL 8.0.